### PR TITLE
add menu to navbar and refactor code

### PR DESCRIPTION
**Pull request by Mahmoudzaid2994**
### Description
Add menu to navbar and refactor code

- Add menu to Navbar
- Add Role page to Admin layout and refactor some styles
- Refactor some styles at Sidebar
- Made some changes at routes

### CODE REVIEW

File: src/routes.js [hunk 1]
```diff
@@ -43,27 +43,19 @@ const routes = [
...
-  {
-    component: lazy(() => import('pages/Listings/CreateListing')),
+    component: lazy(() => import('pages/Listing/CreateListing')),
     label: 'create-listing',
-    path: '/listings/create',
+    path: '/listing/create',
     title: 'Create Listing',
   },
   {
     component: lazy(() => import('pages/ListingSearch')),
```
Comment (AhmadAlShwaiki) [inline src/routes.js:52]:
```suggestion
    component: lazy(() => import('pages/Listing/Search')),
```

File: src/routes.js [hunk 2]
```diff
@@ -43,27 +43,19 @@ const routes = [
...
-    // this is only for testing purposes
-    adminLayout: true,
-    component: lazy(() => import('pages/TestSideBar')),
-    label: 'test-side-bar',
-    path: '/test-side-bar',
-    title: 'Test Sidebar',
-  },
-  {
-    component: lazy(() => import('pages/Listings/CreateListing')),
+    component: lazy(() => import('pages/Listing/CreateListing')),
```
Comment (AhmadAlShwaiki) [inline src/routes.js:46]:
```suggestion
    component: lazy(() => import('pages/Listing/Create')),
```

**Review by AhmadAlShwaiki — APPROVED** [review]
Minor suggestions, looks good!

**Review by AseelRousan-op — APPROVED** [review]

File: src/routes.js (hunk 1 above)
Comment (Mahmoudzaid2994) [reply src/routes.js]: forget to change it.
changed now, thanks

**Pull request merged**